cloud/services/publicloadbalancers/publicloadbalancers.go

```diff
@@ -33,6 +33,7 @@ import (
 type Spec struct {
 	Name         string
 	PublicIPName string
+	Role         string
```

Comment:
@jsturtevant please review if you can as these changes will affect #646 (positively, I hope)

Reply:
Looks like a nice improvement 👍